pages/about.html

```diff
@@ -4,9 +4,43 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>✨About Me✨</title>
+  <link rel="stylesheet" href="../styles/style.css">
+  
 </head>
+<header>
+  <h1>K. Buslach </h1>
+  <h1>ADA C17 Sharks 🦈</h1>
+</header>
+<div class="main">
+<div class="navigate">
+  <ul>
+  <nav>
+    <li><a href="../index.html">Home</a></li>
+    <li><a href="/about.html">About me</a></li>
+    <li><a href="/portfolio.html">Portfolio</a></li>
+  </nav>
+  </ul>
+</div>
```

Comment:
Usually we put the nav element nested in the header element, and the header element can go in the body element as well.

We should also avoid using divs since they're not semantic when we don't need them. Also, it makes more sense to have nav be there parent element to an unordered list. Here using a nav element is sufficient.

The revised HTML would be something like:
```HTML
<body>
  <header>
      <h1>K. Buslach </h1>
      <h1>ADA C17 Sharks 🦈</h1>
          <nav>
            <ul>
                <li><a href="../index.html">Home</a></li>
                <li><a href="/about.html">About me</a></li>
                <li><a href="/portfolio.html">Portfolio</a></li>
            </>ul
          </nav>
      </header>
</body>